### PR TITLE
Streamline docs site

### PR DESCRIPTION
**Pull request by jmandel**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,13 +1,361 @@
-Copyright 2015 Boston Children's Hospital
+Copyright 2020 Boston Children's Hospital
```
Comment (vlad-ignatov) [inline LICENSE]: Feedback from Ken:

Change first line of the license file to read: © Copyright 2020 Computational Health Informatics Program, Boston Children’s Hospital, Boston, MA.

Comment (jmandel) [reply LICENSE]: Updated, thanks!

**Comment by jmandel** [comment]
Thanks for the review. I made the license change m you requested, and will be great if you can make the other after we merge.

**Comment by vlad-ignatov** [comment]
Thanks Josh! I'll make the other changes.

**Pull request merged**